sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceAggregatePushDownSuite.scala

```diff
@@ -261,6 +261,63 @@ trait FileSourceAggregatePushDownSuite
     }
   }
 
+  test("aggregate with partition group by can be pushed down") {
```

Comment:
nit: add JIRA number

Reply:
I want it to be consistent with the rest of the tests, so didn't add a jira number